src/views/Patient/MedReqDropDown/MedReqDropDown.tsx

```diff
@@ -1,250 +1,357 @@
-                                        {selectedMedicationCard?.medicationCodeableConcept?.coding?.[0].display?.split(' ')[0]}
-                                    </Typography>
-                                    <Typography variant='h6' sx={{ bgcolor: 'text.disabled', color: 'white', textAlign: 'center' }} color='textSecondary' gutterBottom>
-                                        {selectedMedicationCard?.medicationCodeableConcept?.coding?.[0].display}
-                                    </Typography>
-                                    {etasu_status_enabled && (
-                                        <Button variant='contained' onClick={handleOpenCheckETASU}>Check ETASU</Button>
-                                    )}
-                                    {pharmacy_status_enabled && (
-                                        <Button variant='contained' onClick={handleOpenCheckPharmacy}>Check Pharmacy</Button>
-                                    )}
-                                </CardContent>
-                            )}
-                        </Card>
-                        {selectedOption ?
-                            <Card>
-                                <CardContent>
-                                    <CdsHooksCards cards={hooksCards} client={client}></CdsHooksCards>
-                                </CardContent>
-                            </Card>
-                            : <p></p>}
-                    </div>
-                </div>
-            </Box >
-            <Modal
-                open={showEtasu}
-                onClose={handleCloseCheckETASU}
-            >
-                <Box sx={modal_style}>
-                    <EtasuStatus patient={patient} medication={selectedMedicationCard} update={showEtasu}></EtasuStatus>
-                </Box>
-            </Modal>
-            <Modal
-                open={showPharmacy}
-                onClose={handleCloseCheckPharmacy}
-            >
-                <Box sx={modal_style}>
-                    <PharmacyStatus patient={patient} medication={selectedMedicationCard} update={showPharmacy}></PharmacyStatus>
-                </Box>
-            </Modal>
+    if (patient && patient.id && userId && selectedMedicationCardBundle) {
+      const hook = new OrderSign(patient.id, userId, {
+        resourceType: 'Bundle',
+        type: 'batch',
+        entry: [selectedMedicationCardBundle]
+      });
+      const tempHook = hook.generate();
+
+      hydrate(getFhirResource, example.prefetch, tempHook).then(() => {
+        setCDSHook(tempHook);
+      });
+    }
+  }, [selectedMedicationCardBundle]);
+
+  useEffect(() => {
+    if (patient && practitioner && selectedMedicationCardBundle) {
+      setSendRxEnabled(true);
+    } else {
+      setSendRxEnabled(false);
+    }
+  }, [patient, practitioner, selectedMedicationCardBundle]);
```

Comment:
The ETASU and Pharmacy status checks can be disabled with an environment variable setting. Should we do the same thing with the send rx button? Is there a use case that we would not want to send the Rx from the app but it would instead be sent by the EHR?